examples/utils/initialize_settings.py

```diff
@@ -89,5 +122,6 @@ def execute():
         set_notebook_environment(environment_variables_config)
 
 
-execute()
-os.environ.update({"is_setup_executed": "True"})
+if __name__ == "__main__":
```

Comment:
Minor thing, initialize_settings.py isn't being executed directly anymore in notebooks, we could remove it altogether

Reply:
@mrakitin - should we remove it?

Reply:
Thanks for your review, @chu-k!

I defined a couple of functions that are used in CLI and in some notebooks, but not the old `execute()` one. I can move those to `__init__.py` if people are OK with it, and then the `initialize_settings.py` can be removed.

Reply:
Let's remove unused functions and rename this file to `notebook.py` maybe?